examples/qm9/train_qm9.py

```diff
@@ -71,14 +69,33 @@ def __call__(self, x0, x1):
         if isinstance(x1, Variable):
             x1 = cuda.to_cpu(x1.data)
         if self.scale == 'standardize':
-            scaled_x0 = self.ss.inverse_transform(cuda.to_cpu(x0))
-            scaled_x1 = self.ss.inverse_transform(cuda.to_cpu(x1))
+            scaled_x0 = self.ss.inverse_transform(x0)
+            scaled_x1 = self.ss.inverse_transform(x1)
             diff = scaled_x0 - scaled_x1
         elif self.scale == 'none':
-            diff = cuda.to_cpu(x0) - cuda.to_cpu(x1)
+            diff = x0 - x1
```

Comment:
It was originally `cuda.to_cpu(x0) - cuda.to_cpu(x1)`, to handle cupy array.
If you remove `cuda.to_cpu`, does it work on GPU??

Reply:
I think above line should be

```
if isinstance(x0, Variable):
    x0 = x0.data
if isinstance(x1, Variable):
    x1 = x1.data
```
to convert Variable to array (numpy array or cupy array)
and then we should call `cuda.to_cpu()` outside of this if block.